src/docs/metrics/development-activity/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Developer Activity metric
```

Comment:
date field (?)

Reply:
Do we really need dates? Should I bump all dates of the files I've touched?